hlapipeline/utils/astrometric_utils.py

```diff
@@ -420,13 +420,25 @@ def build_source_catalog(image, refwcs, **kwargs):
         Specify whether or not to write out a separate catalog file for all the
         sources found in each chip.  Default: None (False)
 
+    threshold : float, optional
+        This parameter controls the threshold used for identifying sources in
+        the image relative to the background RMS in much the same way that
+        the 'threshold' parameter in 'tweakreg' works.
+        Default: 1000.
+
+    fwhm : float, optional
+        FWHM of the expected sources from the image, comparable to the
+        'conv_width' parameter from 'tweakreg'.  Objects with FWHM closest to
+        this value will be identified as sources in the catalog. Default: 3.0.
+
```

Comment:
Could you add units to these items please?  I have been finding items I expect to be in arcseconds are often in pixels.

Reply:
This has been addressed in commit [6c6fd2b](https://github.com/spacetelescope/hlapipeline/pull/27/commits/6c6fd2b2e45ec572075931d1d8e0d428f5bf4560)